ui/src/layouts/chats/presentation/messages/mod.rs

```diff
@@ -5,6 +5,7 @@ use std::{
 };
 
 use dioxus::prelude::{EventHandler, *};
+use arboard::Clipboard;
```

Comment:
```suggestion
use dioxus::prelude::{EventHandler, *};
```


to fix the linter issue 🔨 

![image](https://github.com/Satellite-im/Uplink/assets/29093946/94fa22d4-f69f-47b6-94c4-e7b52f996708)